spec/all_pages_spec.rb

```diff
@@ -33,6 +37,17 @@ def get_locale(path)
         expect(doc).to properly_escape_html
       end
 
+      it 'links consistently' do
+        doc.css('a').each do |a|
+          next if a[:href].start_with?('#')
+          uri = URI(a[:href])
+          next if external_link?(uri)
+          expect(uri).to be_https_scheme, "expected https, got:\n\n#{a.to_html}"
+          expect(uri).to be_plain_host, "expected plain host, got:\n\n#{a.to_html}"
+          expect(uri).to have_trailing_slash, "expected trailing slash, got:\n\n#{a.to_html}"
+        end
```

Comment:
let's wrap this in `aggregate_failures` so we see all the errors at once when there's multiple?

```suggestion
        aggregate_failures do
          doc.css('a').each do |a|
            next if a[:href].start_with?('#')
            uri = URI(a[:href])
            next if external_link?(uri)
            expect(uri).to be_https_scheme, "expected https, got:\n\n#{a.to_html}"
            expect(uri).to be_plain_host, "expected plain host, got:\n\n#{a.to_html}"
            expect(uri).to have_trailing_slash, "expected trailing slash, got:\n\n#{a.to_html}"
          end
        end
```

Reply:
Nice, didn't know this was a thing. Thanks.

Reply:
Updated in 8abb300.